pyapp/cookiecutter.json

```diff
@@ -25,5 +25,11 @@
     "__hdev_project_type": "application",
     "__ignore__": "",
     "__target_dir__": "",
-    "_jinja2_env_vars": {"lstrip_blocks": true, "trim_blocks": true}
+    "_jinja2_env_vars": {"lstrip_blocks": true, "trim_blocks": true},
+    "__parallel_unit_tests": false,
```

Comment:
New `cookiecutter.json` to enable running a project's unit tests in parallel.

We're not going to want this for all projects: it actually makes the tests slower for projects that don't have many tests, it's only faster for larger projects. Also, the project has to have its DB and Elasticsearch stuff setup to work in parallel which not all our projects do yet.